inc/planning.class.php

```diff
@@ -1637,14 +1573,45 @@ static function updateEventTimes($params = []) {
             }
 
             if (!$abort) {
-               $update = ['id'   => $params['items_id'],
-                           'plan' => ['begin' => $params['start'],
-                                           'end'   => $params['end']]];
+               $update = [
+                  'id'   => $params['items_id'],
+                  'plan' => [
+                     'begin' => $params['start'],
+                     'end'   => $params['end']
+                  ]
+               ];
 
                if (isset($item->fields['users_id_tech'])) {
                   $update['users_id_tech'] = $item->fields['users_id_tech'];
                }
 
+               // manage moving event between resource (actors)
+               if (isset($params['new_actor_itemtype'])
+                   && isset($params['new_actor_items_id'])) {
+
+                  $new_actor_itemtype = strtolower($params['new_actor_itemtype']);
+
+                  // reminders don't have group assignement for planning
+                  if (!($new_actor_itemtype === 'Group'
```

Comment:
`$new_actor_itemtype` is lowered.
```suggestion
                  if (!($new_actor_itemtype === 'group'
```